examples/Makefile

```diff
@@ -221,6 +230,9 @@ endif
 ifeq ($(PLATFORM),PLATFORM_RPI)
     CFLAGS += -std=gnu99
 endif
+ifeq ($(PLATFORM),PLATFORM_DRM)
+    CFLAGS += -std=gnu99 -DEGL_NO_X11
```

Comment:
EGL_NO_X11 could also be added directly to the source code, rlgl doesn't depend on X11 data structures after all.

Reply:
To be honest I'm not sure why this didn't came up earlier. The EGL header `eglplatform.h` by default pulls in `Xlib.h` (which seems wrong for me, but it does anyways) which pulls in another X header that declares a structure `Font` and which in turn conflicts with raylib's structure `Font`. This leads me to add EGL_NO_X11 to prevent EGL from including the X11 headers.
On the other hand it seems that EGL and X11 is possible so maybe PLATFORM_DRM could be merged with PLATFORM_DESKTOP in future, so I decided first to not change/implement any other than the DRM related code.